content/blog/sleep/index.md

```diff
@@ -0,0 +1,78 @@
+
+Adults need from 7 to 9 hours of sleep per day and the average American is sleeping **6.5 hours** during the week. This used to be **7.9 hours** in 1942.
+
+There are numbers of important processes happening during the night like muscle repair and grow as well as a body cleanse. There is a sewage system in the brain cleaning the brain of metabolic toxins that we accumulated during the day.
+
+> Like humans, most animals sleep for at least a few hours. Dolphins are interesting because they are conscious breathers and in order to not drown, they need to go to the surface and breathe. They float and shut half of its brain during sleep and keep the other half active to keep the basic body functions active.
+
+`youtube: https://www.youtube.com/watch?v=1ZSzI5mHwS8`
+
+Let's explore what affects sleep.
+
+### What is a circadian rhythm?
+
+It's a 24-hour internal clock, a regulator for sleepiness and wakefulness. When we feel the most energized can vary a bit from person to person, but for most adults, there is a noticeable drop in energy from 1 to 3 in the afternoon and from 2 to 4 at night.
+
+Outside factors like lightness and darkness can also have an impact on the internal clock. The brain sends a signal to the body to release melatonin which makes you sleepy.
+
+### Effects of sleep deprivation
+
+Human beings are the only species that deprive themselves of sleep and we are not wired to compensate for the lack of sleep. Sleep is not like a bank. We can't accumulate a debt and hope to pay it off on the weekend.
+
+![Airport sleep](./airport-sleep.jpg)
+
+Effects of sleep deprivation can show:
+
+- **Worsened brain performance**. Getting less than 6 hours of sleep will drop the time to physical exhaustion by 30%.
+
+- **Lower muscle strength**. The less sleep you have less muscle strength, as well as jump height and running speed decrease.
+
+- **Lack of testesterone**. Men who sleep only 5 to 6 hours a night will have a level of testosterone 6-10 years their senior.
+
+Long term lack of sleep is also connected to diseases, one of them is Alzheimer's disease. This may come as a shocker, but drowsy, sleep deprived driving kills more people on the roads than alcohol or drugs combined.
+
+### How to improve sleep?
+
+Here are some tips to help you with night's sleep:
+
+- **Keep a regular sleep schedule**. Go to bed at the same time every day, try to do it for weekends too. This way, you'll train the body to fall asleep at the same time.
+
+- **Avoid screen time before bed**. Blue light, emitted from mobile phones can postpone melatonin production, which is needed for the body to feel sleepy. One hour of screen time can postpone sleepiness for up to 3 hours.
+
+- **Cool, clean and quiet bedroom**. Bedroom temperature should be between 60 and 67 degrees (15 to 20 degrees Celsius). Eliminate any noises that can disturb sleep and blackout any sources of light from outside lamps or electronic devices.
```

Comment:
_between 60 and 67 degrees_ 

you should define the scale (probably Fahrenheit) as you did for the Celsius.